models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -1,17 +1,22 @@
-from WorkspaceProvider import WorkspaceProvider
+from workspace_provider import WorkspaceProvider
 from mantid.simpleapi import mtd,Load,DeleteWorkspace,Load,GroupWorkspaces,RenameWorkspace,SaveNexus
 
 class MantidWorkspaceProvider(WorkspaceProvider):
+
     def getWorkspaceNames(self):
```

Comment:
I have changed the first functions name, since this module is such a thin wrapper around mantid i have made the function names as similar as possible to the mantid algorithms, should i make the names PEP compliants?